variables.sh

```diff
@@ -2,12 +2,12 @@
 
 DOCKER_REPO=${DOCKER_REPO}
 NAMESPACE=${NAMESPACE:-namely}
-GRPC_VERSION=${GRPC_VERSION:-1.31}
-GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.31}
+GRPC_VERSION=${GRPC_VERSION:-1.32}
+GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.32}
 GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
 GO_VERSION=${GO_VERSION:-1.14}
 ALPINE_VERSION=${ALPINE_VERSION:-3.12}
-BUILD_VERSION=${BUILD_VERSION:-2}
+BUILD_VERSION=${BUILD_VERSION:-1}
```

Comment:
```suggestion
BUILD_VERSION=${BUILD_VERSION:-0}
```

Reply:
again, this PR should go after #205 or #207, depends on which way we will choose.